onnxruntime/core/providers/cpu/tensor/tile.cc

```diff
@@ -46,11 +46,21 @@ Status Tile<float>::Compute(OpKernelContext* ctx) const {
   // Calculate the shape of the output tensor
   auto* repeats = repeats_tensor.template Data<int64_t>();
   std::vector<int64_t> output_dims = input_tensor.Shape().GetDims();
-  for (auto axis = 0; axis < input_tensor.Shape().NumDimensions(); axis++)
+  bool returnEmptyTensor = false;
```

Comment:
A simpler way is : call TensorShape::Size() function, and test if the result is 0. It could make your code a bit shorter.

Reply:
thanks for the tip!